securedrop/securedrop-admin

```diff
@@ -310,7 +319,12 @@ if __name__ == "__main__":
 
     parse_install = subparsers.add_parser('install',
                                           help=install_securedrop.__doc__)
+
     parse_install.set_defaults(func=install_securedrop)
+    parse_install.add_argument(
+        '-t', '--tags', action='append',
+        default=[],
+        help="Run only configuration matching specified Ansible tags")
```

Comment:
Question: do we want admins to know what Ansible is? We should try to abstract away that we're running an Ansible playbook under the hood. For the purpose of this feature, they just need to roughly know what the possible tags are and what they do. In lieu of a SecureDrop admin option like `securedrop-admin list-tags` (running `--list-tags`) we can just list this in the documentation, along with a sentence or so describing what running with that tag will do.

Reply:
Agree that we shouldn't have to expose ansible to the admins.